src/test/scala/io/iohk/ethereum/ledger/BlockRewardSpec.scala

```diff
@@ -51,6 +51,37 @@ class BlockRewardSpec extends FlatSpec with Matchers with MockFactory {
     afterRewardWorldState.getGuaranteedAccount(Address(block.body.uncleNodesList(1).beneficiary)).balance shouldEqual ommerFiveBlocksDifferenceReward
   }
 
+  "Reward Calculation" should "be calculated correctly after byzantium fork" in new TestSetup {
+    val block: Block = sampleBlockAfterByzantium(validAccountAddress)
+    val afterRewardWorldState: InMemoryWorldStateProxy = ledger.payBlockReward(block, worldState)
+    val address = Address(block.header.beneficiary)
+    val beforeExecutionBalance: BigInt = worldState.getGuaranteedAccount(address).balance
+    afterRewardWorldState.getGuaranteedAccount(address).balance shouldEqual beforeExecutionBalance + afterByzantiumNewBlockReward
+  }
+
+  "Reward Calculation" should "be calculated correctly if ommers are included in block after byzantium fork " in new TestSetup {
+    val block: Block = sampleBlockAfterByzantium(validAccountAddress4, Seq(validAccountAddress5, validAccountAddress6))
+
+    val minerAddress = Address(block.header.beneficiary)
+    val ommer1Address = Address(block.body.uncleNodesList.head.beneficiary)
+    val ommer2Address = Address(block.body.uncleNodesList(1).beneficiary)
+
+    val afterRewardWorldState: InMemoryWorldStateProxy = ledger.payBlockReward(block, worldState)
+
+    val beforeExecutionBalance1: BigInt = worldState.getGuaranteedAccount(minerAddress).balance
+    val beforeExecutionBalance2: BigInt = worldState.getGuaranteedAccount(ommer1Address).balance
+    val beforeExecutionBalance3: BigInt = worldState.getGuaranteedAccount(ommer2Address).balance
+
+    afterRewardWorldState.getGuaranteedAccount(minerAddress).balance shouldEqual (beforeExecutionBalance1 + afterByzantiumNewBlockReward)
+
+    // new_uncle_reward = (8 - block.number - uncle.number) * new_block_reward / 8
+    val wei: BigInt = BigInt(10).pow(18) * 3
```

Comment:
you can set it to `blockchainConfig.monetaryPolicyConfig.firstEraReducedBlockReward`